tests/conftest.py

```diff
@@ -73,3 +86,20 @@ def main_page_car_hire_tab(browser):
 @pytest.fixture(scope="module")
 def main_page_search_hotels_tab(browser):
     return MainPageSearchHotelsTab(browser=browser, url=browser.current_url)
+
+
+@pytest.fixture(scope="module")
+def assertions(browser):
+    return Assertions(browser=browser)
+
```

Comment:
I think we should initialize it as self.assertions in BasePage such as ElementInteractions

Reply:
up to u